src/App.jsx

```diff
@@ -335,61 +308,27 @@ function App() {
 						)}
 					</Route>
 
+					{/* Help Center */}
+					<Route path='/returns-calculator'>
+						<ReturnsCalculator
+							colorMode={colorMode}
+							currency='KSM'
+							validatorData={validatorData}
+							onEvent={handleChildTabEvent}
+						/>
+					</Route>
 					{/* Help Center */}
 					<Route path='/help-center'>
 						<HelpCenter />
 					</Route>
-					{/* Suggested Validators */}
+					{/* Suggested Validators*/} 
 					<Route path='/suggested-validators'>
 						<SuggestedValidators
 							colorMode={colorMode}
-							returns={1.43678534556}
-							budget={3000}
+							returns={parseFloat(suggValidatorsData.expectedReturns) || 0}
+							budget={parseFloat(suggValidatorsData.budget) || 0}
 							currency='KSM'
```

Comment:
Preference for global variable over hard-coding currency

Reply:
Done.